src/hapi/pipelines/database/sector.py

```diff
@@ -23,17 +24,16 @@ def __init__(
     ):
         super().__init__(session)
         self._datasetinfo = datasetinfo
-        self.data = {}
-        self._sector_map = sector_map
+        self.data = sector_map
         self.pattern_to_code = {}
 
     def populate(self):
         logger.info("Populating sector table")
 
         def parse_sector_values(code: str, name: str):
             if code != "intersectoral":
-                self.data[name] = code
-                self.data[code] = code
+                self.data[normalise(name)] = code
+                self.data[normalise(code)] = code
                 pattern = code.lower().replace("-", "_")
                 pattern = TagPattern.parse(f"#*+{pattern}")
                 self.pattern_to_code[pattern] = code
```

Comment:
Are these three lines needed anymore?

Reply:
Good spot - pattern_to_code is not used anywhere